template/src/Autovis/Controls.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import "./controls.css";
+import pad from './pad';
+
+
+const printTime = (currentTime) => {
+  const minutes = Math.floor(currentTime / 60);
+  const seconds = Math.floor(currentTime % 60);
+  return `${pad(minutes, 2)}:${pad(seconds, 2)}`;
+}
+
+const convertPositionToSeconds = (e, totalSeconds) => {
```

Comment:
i think this makes a bit more sense if it's not bound to totalSeconds. i'd just return the seekRatio and multiply the result when it gets used down below:

`onSeek(getEventXPositionRatio(e) * duration)`